content/docs/components/hyperparameter.md

```diff
@@ -4,25 +4,91 @@ description = "Using Katib to tune your model's hyperparameters"
 weight = 5
 +++
 
-## Using Katib
+Hyperparameter Tuning on Kubernetes.
+This project is inspired by [Google vizier](https://static.googleusercontent.com/media/research.google.com/ja//pubs/archive/bcb15507f4b52991a0783013df4222240e942381.pdf). Katib is a scalable and flexible hyperparameter tuning framework and is tightly integrated with kubernetes. Also it does not depend on a specific Deep Learning framework (e.g. TensorFlow, MXNet, and PyTorch).
+
+## Installing Katib
+
+For running Katib jobs, you have to install necessary packages.
+
+In your Ksonnet app root, run the following
 
-Currently we are using port-forwarding to access the katib services.
-kubernetes version 1.9~
 ```
-kubectl -n kubeflow port-forward svc/katib-ui 8000:80
+export KF_ENV=default
+ks env set ${KF_ENV} --namespace=kubeflow
+ks registry add kubeflow github.com/kubeflow/kubeflow/tree/master/kubeflow
 ```
-~1.8
+
+### TF operator
+
+For installing tf operator, run the following
+
+```
+ks pkg install kubeflow/tf-training
+ks pkg install kubeflow/common
+ks generate tf-job-operator tf-job-operator
+ks apply ${KF_ENV} -c tf-job-operator
+```
+
+### Pytorch operator
+For installing pytorch operator, run the following
+
+```
+ks pkg install kubeflow/pytorch-job
+ks generate pytorch-operator pytorch-operator
+ks apply ${KF_ENV} -c pytorch-operator
 ```
-kubectl get pod -n kubeflow  # Find your katib-ui pods
-kubectl port-forward -n kubeflow [katib-ui pod] 8000:80 &
+
+### Katib
+
+Finally, you can install Katib
+
+```
+ks pkg install kubeflow/katib
+ks generate katib katib
+ks apply ${KF_ENV} -c katib
 ```
-## Creating a Study Job
+
+If you want to use Katib not in GKE and you don't have StorageClass for dynamic volume provisioning at your cluster, you have to create persistent volume to bound your persistent volume claim.
+
+This is yaml file for persistent volume
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: katib-mysql
+  labels:
+    type: local
+    app: katib
+spec:
+  capacity:
+    storage: 10Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: /data/katib
+```
+
+Create this pv after deploying Katib package
```

Comment:
s/pv/persistent volume (PV)

Because it's best to spell out an acronym the first time we use it, for readers who haven't seen it before.